app/libs/confy/reducers/wizardReducer.js

```diff
@@ -0,0 +1,17 @@
+import {getDefaultModel} from "../fields"
+const defaultState = {
+    model: undefined,
```

Comment:
I wanted it to be some kind of easy to read state definition, but now I think you are right, it can be removed.